Fails to build on Debian Jessie

After commenting out osquery/tools/provision/debian.sh
```
  #package ruby1.8-dev
```

during the ```make deps``` step, I get the error 
```
[+] aws_sdk has not been extracted. extracting...
/tmp/osquery-provisioning/aws-sdk-cpp-0.10.6/build /tmp/osquery-provisioning
CMake Error at CMakeLists.txt:19 (cmake_minimum_required):
  CMake 3.1.2 or higher is required.  You are running version 3.0.2


-- Configuring incomplete, errors occurred!
Makefile:88: recipe for target 'deps' failed
make: *** [deps] Error 1
```


Looks like Jessie is only has a package for 3.0.2. Boohoo :(

--- a/tools/provision/debian.sh
+++ b/tools/provision/debian.sh
@@ -27,7 +27,6 @@ function main_debian() {
   package python-pip
   package python-dev
   package ruby-dev
-  package ruby1.8-dev
   package libffi-dev
   package rubygems
   package gcc
@@ -52,12 +51,11 @@ function main_debian() {
   package clang
 
   if [[ $DISTRO == "wheezy" ]]; then
-    install_cmake
+    package ruby1.8-dev
     # thrift requires automate 1.13 or later
     remove_package automake
     install_automake
   elif [[ $DISTRO == "jessie" ]]; then
-    package cmake
     package automake
   fi
 
@@ -69,6 +67,7 @@ function main_debian() {
     gem_install fpm
   fi
 
+  install_cmake
   install_boost
   install_gflags
   install_glog

